static-site/content/blog/2025-01-06-human-metapneumovirus-analysis-and-resources.md

```diff
@@ -0,0 +1,49 @@
+---
+author: "Isabel Joia, Richard Neher"
+date: "2025-01-05"
```

Comment:
Small reminder that the date should be updated before merging. Merging will automatically put it on https://next.nextstrain.org. I (or another dev) can then push it to https://nextstrain.org on the same day.

```suggestion
date: "2025-01-06"
```

Reply:
#1103